html/manifest/icons.json

```diff
@@ -0,0 +1,58 @@
+{
+  "html": {
+    "manifest": {
+      "icons": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/Manifest",
+          "support": {
+            "chrome": {
+              "version_added": null
+            },
+            "chrome_android": {
+              "version_added": true
+            },
+            "edge": {
+              "version_added": null
+            },
+            "edge_mobile": {
+              "version_added": null
+            },
+            "firefox": {
+              "version_added": null
+            },
+            "firefox_android": {
+              "version_added": "53",
+              "notes": "By default, used for as the source for \"Add to home screen\" (see bug 1234558). Experimental support for this feature is available behind the boolean flag manifest.install.enabled in about:config."
```

Comment:
Can you turns this into a `flags` object?
https://github.com/mdn/browser-compat-data/blob/master/schemas/compat-data-schema.md#flags